datasets/dataset_imdb.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+IMDB dataset.
+inspired from:
+https://keras.io/examples/nlp/bidirectional_lstm_imdb/
+"""
+
+import keras
+from keras.utils import to_categorical
+from keras.preprocessing.sequence import pad_sequences
+from keras import models, layers
+from keras.datasets import imdb
+
+# Definition of dataset-specific parameters 
+maxlen = 200 # We only keep the 200 first words of the reviews
+out_of_vocabulary = 2
```

Comment:
Let's provide an explicative comment about that one (otherwise no one can understand what this is :)!

Also, I think there is PEP8 thing for inline comments: two whitespaces before the `#`

Reply:
`Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.` yes, it's done